src/client/package.json

```diff
@@ -22,23 +22,20 @@
   "scripts": {
```

Comment:
I ran install again and pushed, hopefully that resolves. I'm running npm v 9.1.1, perhaps you're running an earlier version with a different lockfile format? In any case we use `npm ci` in the build, so that should follow what's in the lockfile.

Reply:
yep, all good now 👍🏻